src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationapi/data/model/UserAuthorizationCode.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.authorizationapi.data.model
+
+import jakarta.persistence.AttributeConverter
+import jakarta.persistence.Column
+import jakarta.persistence.Converter
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import uk.gov.justice.digital.hmpps.authorizationapi.service.AuthSource
+import java.time.Instant
+
+@Entity
+@Table(name = "oauth2_user_authorization_code")
+data class UserAuthorizationCode(
+  @Id
+  val id: String,
+
+  val userName: String,
```

Comment:
should this be `username` (without the capital N) for consistency?

Reply:
> should this be `username` (without the capital N) for consistency?

updated